### ISSUE TITLE
Error `Unable to encode the output with cp1252 encoding.` is shown when logging in to a subscription which name contains Japanese characters and on Windows runner

### ISSUE THREAD

**Issue by horihiro** [Azure/login#216]
When trying to log in with a subscription which name contains Japanese characters by the following simple workflow, the logging in is failed with the encoding error like the following screenshot.

Workflow:
```yaml
name: Azure Login Test

on:
  workflow_dispatch:

jobs:
  login:
    runs-on: windows-latest # using Windows runner
    steps:
    - uses: azure/login@v1
      with:
        creds: ${{ secrets.AZURE_CREDENTIALS }}
```

Error screenshot:
![image](https://user-images.githubusercontent.com/4566555/161893707-50277d56-1835-45a6-be8b-4d16f41ee6ca.png)

**Comment by horihiro** [comment]
@BALAGA-GAYATRI could you please take a look at this or assign anyone?

**Comment by BALAGA-GAYATRI** [comment]
@horihiro we shall look into this issue and let you know.

**Comment by horihiro** [comment]
Hello @BALAGA-GAYATRI 
I did addional check by the following job.

```yaml
jobs:
  login:
    runs-on: windows-latest
    steps:
    - run: |
        az login --service-principal -u ${{ secrets.AZURE_CLIENT_ID }} -p ${{ secrets.AZURE_CLIENT_SECRET }} --tenant ${{ secrets.AZURE_TENANT_ID }}
```

Then, the job was succeeded, even though the same warning message `WARNING: Unable to encode the output with cp1252 encoding. Unsupported characters are discarded.` was shown.
![image](https://user-images.githubusercontent.com/4566555/163898966-49d49098-c638-4f82-896c-e7e4f55cf8d2.png)


From this result, it seems that the `azure/login` action and `@action/exec` module which is used in this action handle this warning as an error.
Could you please check this point?

**Comment by BALAGA-GAYATRI** [comment]
Sure. Will check this out and let you know. Thanks for the extra insights.

**Comment by BALAGA-GAYATRI** [comment]
Fixed this bug in [v1.4.4](https://github.com/Azure/login/releases/tag/v1.4.4) and updated `azure/login@v1` with the latest release.

**Comment by horihiro** [comment]
Thanks @BALAGA-GAYATRI !
